api/src/components/publication/controller.ts

```diff
@@ -136,6 +137,15 @@ export const create = async (
 
         const publication = await publicationService.create(event.body, event.user, doi);
 
+        // add references to the publication
+        event.body.references?.map(async (item) => {
```

Comment:
Is this change still needed? I think we can remove this since we have dedicated endpoint for references now.